packages/styles/README.md

```diff
@@ -33,6 +33,13 @@ application:
 import '@carbon/ibmdotcom-styles';
 ```
 
+For Carbon component styles to properly import, the following should be added to
+your environment variables:
+
+```
+SASS_PATH=node_modules:src
+```
```

Comment:
@kennylam wondering if this is enough context for adopters or if we should provide a more detailed explanation around this, and if this is an issue on the Carbon side or not that we're trying to do a workaround for?

cc: @RaphaelAmadeu

Reply:
@jeffchew @RaphaelAmadeu Yes, agree a little more detail could be helpful.
 maybe something like 
```suggestion
> 💡 There may be times pathing errors are encountered when importing certain stylesheets. We're working hard to fix
> these, but in the meantime you can add the following to your `.env` file to resolve.

```
SASS_PATH=node_modules:src
```
```

Reply:
@kennylam Thanks for the explanation text, I've added it to my latest PR